README.md

```diff
@@ -32,13 +32,17 @@ Serialization providers are responsible for ABI-driven transaction and action se
 * Gradle Plugin 3.3.0+
 * For Android, Android 6 (Marshmallow)+
 
-This project relies on platform functionality and libraries only present in Android 6+ and the Android NDK. Therefore, any project depending on ABIEOS Serialization Provider with [EOSIO SDK for Java](https://github.com/EOSIO/eosio-java) **must be an Android 6+ project**. If you need support for ABIEOS serialization on server side Java, please see the [Java ABIEOS Serialization Provider](https://github.com/EOSIO/eosio-java-abieos-serialization-provider) project. Other serialization providers, however, can be created to support earlier Android versions or other platforms. If your project requires earlier Android version or alternate platform support, or if you'd like to create a serialization provider and have questions, please reach out to us by [logging an issue](/../../issues/new).
+This project relies on platform functionality and libraries only present in Android 6+ and the Android NDK. Therefore, any project depending on Android ABIEOS Serialization Provider with [EOSIO SDK for Java](https://github.com/EOSIO/eosio-java) **must be an Android 6+ project**.
+
+If you need support for ABIEOS serialization on server side Java, please see the [Java ABIEOS Serialization Provider](https://github.com/EOSIO/eosio-java-abieos-serialization-provider) project.
+
+Other serialization providers can be created to support earlier Android versions or other platforms. If your project requires earlier Android version or alternate platform support, or if you'd like to create a serialization provider and have questions, please reach out to us by [logging an issue](/../../issues/new).
```

Comment:
"If your project requires an earlier Android version....

Reply:
I believe this one can go both ways. If you drop "or alternate platform" and read it, you can see how it could work either way. Another way to read it: "If your project requires earlier-Android-version or alternate-platform support,..."